modules/nodes-pods-autoscaling-creating-cpu.adoc

```diff
@@ -70,6 +70,53 @@ $ oc autoscale dc/<dc-name> \// <1>
 <2> Optionally, specify the minimum number of replicas when scaling down.
 <3> Specify the maximum number of replicas when scaling up.
 <4> Specify the target average CPU utilization over all the pods, represented as a percent of requested CPU. If not specified or negative, a default autoscaling policy is used.
++
+[TIP]
+====
+You can alternatively apply the following YAML to scale based on the percent of CPU:
+
+.Sample `HorizontalPodAutoscaler` object for scaling a deployment config on the percent of CPU utilization
+[%collapsible]
+=====
+[source,yaml]
+----
+kind: HorizontalPodAutoscaler
+apiVersion: autoscaling/v2beta2 <1>
+metadata:
+  name: rails-postgresql-example
+  namespace: aaa
+  uid: abe83eaf-f9a4-4f17-a7de-851438868449
+  resourceVersion: '57294'
+  creationTimestamp: '2021-05-12T17:52:09Z'
+spec:
+  scaleTargetRef: 
+    kind: DeploymentConfig <2>
+    name: rails-postgresql-example <3>
+    apiVersion: apps.openshift.io/v1 <4>
+  minReplicas: 1 <5>
+  maxReplicas: 3 <6>
+  metrics: <7>
+    - type: Resource
+      resource:
+        name: cpu <8>
+        target:
+          type: Utilization <9>
+          averageUtilization: 50 <10>
+----
+<1> Use the `autoscaling/v2beta2` API.
+<2> The kind of object to scale, either `ReplicationController` or `DeploymentConfig`.
```

Comment:
Because this example seems to be for the DeploymentConfig and it is followed by an example for the ReplicationController, can you remove the either-or experience in this callout list and the next one.